.ci/schedule-daily.groovy

```diff
@@ -88,6 +89,54 @@ pipeline {
         )
       }
     }
+    stage('Bump versions observability-test-environments'){
+      matrix {
+        axes {
+          axis {
+            name 'BRANCH'
+            values (
+              'dev',
+              'dev-next',
+              'release'
+            )
+          }
+        }
+        stages {
+          stage('Checkout'){
+            steps {
+              gitCheckout(basedir: '.', branch: env.BRANCH,
+                repo: 'git@github.com:elastic/observability-test-environments.git',
+                credentialsId: "${JOB_GIT_CREDENTIALS}",
+                githubNotifyFirstTimeContributor: false)
+            }
+          }
+          stage('Fetch latest version'){
+            steps {
+              echo 'TBD - Search the latest version'
```

Comment:
my manual process for this 😄 
* Check https://artifacts-api.elastic.co/v1/versions/
* Pick the next [major|minor|patch] snapshot/BC

so we can run it on schedule something like this:
* looking for the next patch [snapshot|BC]
  * if there is not minor check for the next minor
  * if there is not minor check for the next major.
* Check if there is a PR for the update found
  * if there is no PR Create a PR for the update else skip the PR creation